Resources/public/css/countable.css

```diff
@@ -0,0 +1,10 @@
+.error { color: #900;}
```

Comment:
@vanhoog Maybe it's better to use class names specific for this js-counter thing? .error is kind of global. Same for counter-text? Maybe specify it more on this new element?

Reply:
I could refactor it with the new Just CSS naming standard, which i'll do... I'll keep you posted! :)

Reply:
It's important to specify this correctly, maybe wrap it all in ```wearejust-counter``` class? We don't need conflicting classnames